### DISCUSSION TITLE
handover, project continuity 🫴

### DISCUSSION THREAD

**Discussion by n3rada** [Pennyw0rth/NetExec#15] [General]
Although I wasn't a major contributor and perhaps operated more on the periphery as a user of this tool, I felt compelled to share my thoughts with the community..

Recognizing the intrinsic challenges of managing and contributing to open-source projects—given the considerable time and effort they demand—it's comprehensible when contributors decide to reduce or end their involvement. However, However, I believe it is imperative to ensure a smooth transition, akin to a job handover. This is not merely an expectation but a **crucial step rooted in ensuring the sustained success and continuity of one project**.

Presently, several outstanding concerns merit our attention. One notable point of contention is the debate around the project's name. Furthermore, there's an immediate need to address the stewardship of the well-constructed [documention](https://www.crackmapexec.wiki/), where it'll be hosted, by whom etc.

While I haven't been entirely abreast with the entirety of the discussions on various channels—primarily to prioritize my time efficiently—I've been keeping tabs via this discussion thread: https://github.com/Porchetta-Industries/CrackMapExec/discussions/801

Given the aforementioned points, would it be feasible for the community to receive a definitive directive (something else than this [X post](https://twitter.com/mpgn_x64/status/1698990491243606226)) from both the primary maintainers and the erstwhile project leader (@mpgn).This would greatly streamline our collaborative efforts and provide clearer direction for the project's journey ahead.

**Comment by bongobongoland** [comment]
Hello ChatGPT. Still working on a paper for that nugatory peer-reviewed journal of pseudoscience, eh? Handhover? Is that handover + hangover? 

Looks like we have a new name, so the debate is over. The name is... Hangover! It's NetExec...  Which is meh. Whatever. A boring one. Unlike DrShiva!!!

Yes, the docs need an thorough update. I think it would be better to host them here or maybe on readthedocs or something similar instead of hosting them on www.netexec.wiki

@mpgn is done with the project, as far as I know - I've been in touch with him - so I doubt he'll comment.

**Reply by n3rada** [reply]
Hello, 

Knowing how to talk is not always a LLM sir. I let you use the proper tool to check that this is not LLM. I hope you will be more courteous in the future when someone is interested in a project. I'm not here to prowl around shouting in other people's faces. 

As for the mistake in the title, it's called a typo, now corrected. Thanks! 

The  name looks fine in my opinion. I was mentioning it only to start my sentence. Regarding the fact that I'll not comment, that's the main problem. 

Have you ever seen someone with enough dedication leave a project without ensuring its continuity? Especially in our field, which is very community-based.

**Reply by bongobongoland** [reply]
Thank you sir for your interest in NetExec :)

**Comment by Marshall-Hallenbeck** [comment]
@n3rada thanks for the interest! We have decided on a name and are working on renaming code, etc as fits. We are also working on a new, updated Wiki to cover all the awesome changes we've implemented over the past few months. We have a Discord here: https://discord.gg/XqQHd5aU76 to discuss various topics.

As for the vision, we have many different features and refactors planned, so stay tuned for more great content!
